ts/Core/Tooltip.ts

```diff
@@ -1068,15 +1068,16 @@ class Tooltip {
                         });
                     }
 
+                    // Set the stroke color of the box to reflect the point
```

Comment:
This comment seems out of place. It's probably just a remnant of older code, so we can just remove it now.